scheduler/src/cook/mesos/task.clj

```diff
@@ -18,7 +18,7 @@
   (->> resources (map :scalar) (apply +)))
 
 (defmethod combine-like-resources :value-ranges [resources]
-  (->> resources (map :ranges) (apply into)))
+  (->> resources (map :ranges) (apply (comp vec flatten conj))))
```

Comment:
@wyegelwel  Yes, (reduce into) also works (makes the test cases pass) and is definitely more elegant than (comp vec flatten conj).  Thanks @dgrnbrg !